src/main/java/Controllers/RegistrationServlet.java

```diff
@@ -61,6 +61,6 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         request.setAttribute("duplicate_email_error", false);
         request.getRequestDispatcher("register.jsp").forward(request, response);
-        Tables.recreateTables();
+//        Tables.recreateTables();
```

Comment:
Is this supposed to still be commented out?

Reply:
Yes, it is, a button calls it